src/ray/raylet/local_object_manager.cc

```diff
@@ -75,15 +88,37 @@ void LocalObjectManager::WaitForObjectFree(const rpc::Address &owner_address,
   }
 }
 
-void LocalObjectManager::ReleaseFreedObject(const ObjectID &object_id) {
+void LocalObjectManager::ReleaseFreedObject(const ObjectID &object_id,
+                                            const rpc::Address &owner_address) {
   RAY_LOG(DEBUG) << "Unpinning object " << object_id;
   // The object should be in one of these stats. pinned, spilling, or spilled.
   RAY_CHECK((pinned_objects_.count(object_id) > 0) ||
             (spilled_objects_url_.count(object_id) > 0) ||
             (objects_pending_spill_.count(object_id) > 0));
+
+  auto &owners = util::get_ref_or_fail(object_owners_, object_id);
+  auto owner_check = [worker_id = owner_address.worker_id()](const auto &v) {
```

Comment:
Why don't we just use set?

Reply:
That's because owner_address can't be hashed right now. I can add a new hash table for this if you think it's necessary.

Reply:
Yeah I prefer to do that if it is not too complicated. We just need to define how to hash this somewhere right?